go/cmd/cleanup/cleanup_pipelineruns.yaml

```diff
@@ -0,0 +1,13 @@
+# Cleanup old pipelineruns in the ci system
+spec:
+  groups:
+    - Group: "tekton.dev"
+      Version: "v1alpha1"
+      Namespace: "kf-ci"
+      MinAge: "72h"
+      Resource: "pipelineruns"
+    - Group: "tekton.dev"
+      Version: "v1alpha1"
+      Namespace: "kf-ci"
+      MinAge: "72h"
```

Comment:
If the above is difficult. Let's set the MinAge to "168h" i.e. one week. Because anyways the pipeline runs have been there from ages.

Reply:
You can identify pods using labels. The pod's labels include the pipelineRun as well as taskRun etc... Pod logs are preserved in stackdriver and in stackdriver you can search by labels.

```
  labels:
    app.kubernetes.io/managed-by: configmanagement.gke.io
    pipeline: xgboost-notebook
    tekton.dev/pipeline: notebook-test
    tekton.dev/pipelineRun: xgboost-vqsjr
    tekton.dev/pipelineTask: build-image
    tekton.dev/task: notebook-test-builder
    tekton.dev/taskRun: xgboost-vqsjr-build-image-4ppqn

```

Reply:
+1 for storing for a week initially, it will still be easier debugging from pipelinerun resource.

Reply:
@Bobgy Agree. Lets set it appropriately when we automate it per #750 
right now it was pretty aggressive because it was a one time cleanup.